src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -177,7 +177,7 @@ private String getEIBackendSubscriptionAddress() {
     private String getEIRequestURL(HttpServletRequest request) {
         String eiBackendAddressSuffix = request.getServletPath();
         String requestQuery = request.getQueryString();
-        String query = (requestQuery != null && !requestQuery.isEmpty()) ? "?" + requestQuery : "";
+        String query = (requestQuery != null && !requestQuery.isEmpty() && !requestQuery.startsWith("_")) ? "?" + requestQuery : "";
```

Comment:
Why do we get these "?_2342342" stuff in the request? Shouldn't it be fixed in JS code where the request is made?